slatedb/src/db.rs

```diff
@@ -218,6 +218,7 @@ impl DbInner {
                 .await
             {
                 Ok(_) => {
+                    self.oracle.next_wal_id.store(empty_wal_id + 1);
```

Comment:
It feels a little brittle to have this here.. WDYT about putting it in `flush_imm_table`? That's where we update `last_remote_persisted_seq` as well.